docker-compose.yml

```diff
@@ -117,7 +117,7 @@ services:
       - metadata:/var/lib/postgresql/data
 
   sourced-ui:
-    image: srcd/sourced-ui:v0.6.0
+    image: srcd/sourced-ui:latest
```

Comment:
Since `latest` points to the same `v0.6.0`, travis is not really testing the changes from https://github.com/src-d/sourced-ui/pull/285, right? I'd say we can release sourced-ui today and change the version here before merging

Reply:
0.7.0 is now released and available in docker hub :rocket: